package.json

```diff
@@ -8,18 +8,23 @@
         "exampleIngest": "npm run build && node packages/azure-kusto-ingest/example.js",
         "lint": "eslint -c .eslintrc.js --ext .ts .",
         "test": "mocha --parallel",
-        "testNoE2E": "npm run test -- --exclude **/e2eTest.ts",
-        "testData": "npm run test -- --exclude !packages/azure-kusto-data/**",
+        "testE2E": "mocha -- --grep \"E2E Tests\"",
+        "testNoE2E": "npm run test",
+        "testData": "npm run test -- --exclude !packages/azure-kusto-ingest/test/e2eTests/**",
         "testIngest": "npm run test -- --exclude !packages/azure-kusto-ingest/**",
         "testIngestNoE2E": "npm run testNoE2E -- --exclude !packages/azure-kusto-ingest/**",
         "testPipeline": "nyc --reporter lcovonly mocha --reporter mocha-junit-reporter --reporter-options outputs=true",
         "format": "npx prettier --write .",
-        "checkFormat": "npx prettier --check --end-of-line lf ."
+        "checkFormat": "npx prettier --check --end-of-line lf .",
+        "webpack": "npm run build && shx cp packages/azure-kusto-ingest/index.html packages/azure-kusto-ingest/dist-esm/ && webpack serve",
```

Comment:
This command serves webpack, which is fine for development (expect .. not really? does it update well when you change a file? because you only build once), but it won't help in CI/CD (since the webpack server will remain running + it's only dev, not actual release build)